### PR TITLE
Conda build_tutorial improvements

### PR DESCRIPTION
**Pull request by AlbertDeFusco**
1. All three tutorials have been edited to meet the
   recommendations in the Documentation Standards
   Guide and Style Manual

2. Several grammatical issues were fixed in tutorials 1 and 2

3. The postgis tutorial has been completely rewritten.
   The postgis tutorial is now shorter and more focused on writing
   a conda build recipe for a general source code package.

4. The tutorial source files were tested and verified with
   sphinx-build using make html

### CODE REVIEW

**Comment by kerrywatson1** [comment]
This looks great, @AlbertDeFusco! Thanks for your work on this!

Assigning to @electronwill to test and merge.

File: docs/source/build_tutorials/pkgs.rst [hunk 1]
```diff
@@ -115,68 +141,181 @@ Now verify that Pyinstrument installed successfully:
...
 
-4. Optional - Build for a different Python version
---------------------------------------------------
-By default, conda build will build packages for the version of Python installed
-in the root environment. If you'd like to build packages for other versions of
-Python, you can simply use the `--python` flag, followed by a version. For
-example, to explicitly build a version of the Pyinstrument package for Python
-3.3, use
+In order to use these packages they will have to be transferred to other computers and placed in the correct
+``conda-bld/$ARCH`` directory like shown above, where ``$ARCH`` can be ``osx-64``, ``linux-32``, ``linux-64``,
```
Comment (electronwill) [inline docs/source/build_tutorials/pkgs.rst]: like shown above -> as shown above

File: docs/source/build_tutorials/pkgs2.rst [hunk 2]
```diff
@@ -238,39 +177,34 @@ The other two files you need for a build are
 
 These two build files contain all the variables such as for 32-bit or 64-bit architecture (the ARCH
 variable) and the build environment prefix (PREFIX). The two files ``build.sh`` and ``bld.bat`` files must be
-in the same directory as your ``meta.yaml`` file.
+in the same directory as your ``meta.yaml`` file. In this tutorial you're going to make both ``build.sh`` and
+``bld.bat`` so that other users can built the appropriate package for their architecture.
```
Comment (electronwill) [inline docs/source/build_tutorials/pkgs2.rst]: can built -> can build

File: docs/source/build_tutorials/postgis.rst [hunk 3]
```diff
@@ -1,586 +1,286 @@
...
-``proj`` 4.8.0 can be built from source in the same way. It should not have
-non-standard dependencies. I uploaded my package as ``proj``. I include ``-
-proj`` under the build and run requirements in ``meta.yaml`` and add the flag
-``--with-projdir=$PREFIX`` in ``build.sh`` file.
+During execution of the conda-build command the ``$PREFIX`` environment variable is used to refer to the install path
+of conda packages.  We will use ``$PREFIX`` to inform the configure command of the location of the dependent packages
+listed in the build and run requirements of the conda recipe.
 
-At this point the series of dependencies I have assembled is reflected by the conda build script's output:
+In a text edit make a new file called build.sh with the following content in the postgis directory.
```
Comment (electronwill) [inline docs/source/build_tutorials/postgis.rst]: In a text edit -> In a text editor

**Comment by electronwill** [comment]
Much improved, @AlbertDeFusco -- thank you for all your hard work here! I made line notes about three tiny typos, and then it's ready to merge.

**Comment by AlbertDeFusco** [comment]
Thanks, @electronwill

**Pull request merged**